docs/xgc.md

```diff
@@ -60,6 +62,26 @@ Options that change the behavior of the Garbage Collector (GC).
 
 : This option sets the scan mode for GC operations that evacuate objects in the heap (scavenge operations (`gencon`) and copy forward operations (`balanced`)) to breadth first mode. The scan mode reflects the method for traversing the object graph and is also known as *Cheney's algorithm*.
 
+### `classUnloadingKickoffThreshold`
+
+         -Xgc:classUnloadingKickoffThreshold=<value>
+
+: Where `<value>` is equal to the number of class loaders plus the number of anonymous classes that are loaded since the previous class unloading operation.
+
+: This option sets a threshold that is used to start an early concurrent GC cycle due to recent class loading activity. The default value is 80000.
```

Comment:
we can emphasize we are talking about 'concurrent _global_ GC' (since lately we've had concurrent scavenge, as well).